proto/pdpb2.proto

```diff
@@ -0,0 +1,298 @@
+syntax = "proto3";
+package pdpb2;
+
+import "metapb.proto";
+import "eraftpb.proto";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+service PD {
+    rpc Tso(TsoRequest) returns (TsoResponse) {}
+
+    rpc Bootstrap(BootstrapRequest) returns (BootstrapResponse) {}
+
+    // TODO: Carry it by rpc Bootstrap? Since a BOOTSTRAPPED Error is already
+    // definded in ResponseHeader.
+    rpc IsBootstrapped(IsBootstrappedRequest) returns (IsBootstrappedRequest) {}
+
+    rpc AllocId(AllocIdRequest) returns (AllocIdResponse) {}
+
+    rpc GetStore(GetStoreRequest) returns (GetStoreResponse) {}
+
+    rpc PutStore(PutStoreRequest) returns (PutStoreResponse) {}
+
+    rpc StoreHeartbeat(StoreHeartbeatRequest) returns (StoreHeartbeatResponse) {}
+
+    rpc AskSplit(AskSplitRequest) returns (AskSplitResponse) {}
+
+    rpc ReportSplit(ReportSplitRequest) returns (ReportSplitResponse) {}
+
+    rpc GetRegion(GetRegionRequest) returns (GetStoreResponse) {}
+
+    // TODO: Carry it by rpc GetRegion?
+    // Just a matter of a new filed in GetRegionRequest.
+    rpc GetRegionByID(GetRegionByIDRequest) returns (GetRegionResponse) {}
```

Comment:
Carry it by rpc GetRegion? Just a matter of adding a new filed in GetRegionRequest.